src/ListView/PageTableHoverOptions.jsx

```diff
@@ -1,32 +1,54 @@
-import React from "react";
-import { HoverDisclosure, OptionsMenu } from "@sailthru/stui-components";
+import React, { useState } from "react";
+import {
+  HoverDisclosure,
+  ModalWrapper,
+  OptionsMenu,
+} from "@sailthru/stui-components";
+import { DeletePageModal } from "./DeletePageModal";
+
+function PageTableHoverOptions({
+  pageId,
+  pageEditorUrl,
+  pageName,
+  setNotificationType,
+  setNotificationMessage,
+}) {
+  const [displayDeleteModal, setDisplayDeleteModal] = useState(false);
 
-const PageTableHoverOptions = ({ pageId, pageEditorUrl }) => {
   const clickOptions = [
     {
       label: "Edit",
       icon: "fal fa-edit fa-fw",
       iconHover: "fas fa-edit fa-fw",
-      onClick: () => window.open(`${pageEditorUrl}`, "_self"), // TODO see EPT-1713 (add routing?)
+      onClick: () => window.open(`${pageEditorUrl}`, "_self"),
     },
     {
       label: "Delete",
       icon: "fal fa-trash fa-fw",
       iconHover: "fas fa-trash fa-fw",
       onClick: () => {
-        console.log("Deleting", pageId);
-        //         displayModal({
-        // TODO see EPT-1713
-        //         });
+        setDisplayDeleteModal((state) => !state);
```

Comment:
nit-pick: Can this `onClick` be one line like "Edit"?

Reply:
question: Is there a reason this cannot be simplified to this?
```suggestion
        setDisplayDeleteModal(true);
```

Reply:
I kept this just as following the example in STUI
https://stui.design.sailthru.com/?path=/docs/components_design-system-components-modal--as-default

Reply:
non-blocking: The `onDismiss` is using the simplified approach. I find the negating state more confusing than just `true`. But also not a major issue either.
```
onDismiss={() => setDisplayDeleteModal(false)}
```

Reply:
updated this to true but the new line is a `prettier` change